op_crates/websocket/01_websocket.js

```diff
@@ -294,59 +294,78 @@
     }
 
     async #eventLoop() {
-      if (this.#readyState === OPEN) {
```

Comment:
This seems like a behavioral change - is something being fixed here?

Reply:
no behaviour is identical. no fixes, just a cleanup i have been meaning to do for a while

Reply:
Why does this if statement change to a while?

Reply:
as mentioned, it was recursive calls. that if was the guard if to get the next. since i changed to a while loop and the calls are not recursive anymore, it is equivalent. `this.#readyState` gets only changed if the ws closes or errors, so the loop would be exited by that changing